docs/General/documentation-site.md

```diff
@@ -0,0 +1,69 @@
+# Documentation Site
+
+This project exists to consolidate and standardize documentation across DPL
+(Danish Public Libraries) repositories. It's based on [mkdocs
+material](https://squidfunk.github.io/mkdocs-material/) and uses a handy
+[multirepo-plugin](https://github.com/jdoiro3/mkdocs-multirepo-plugin) to import
+markdown and build documentation from other DPL repositories dynamically.
+
+## Adding documentation
+
+### General documentation
+
+Documentation that isn't specific to any particular repository should
+be placed in this [general
+section](https://github.com/danskernesdigitalebibliotek/dpl-docs/tree/main/docs/General/). 
+
+### External documentation
+
+Any repository you want to add to the documentation site, must have a `docs/`
+folder in the project root on github and relevant markdown must be moved inside.
```

Comment:
With this info in mind I suggest reverting `docs/General/documentation-site.md` to `docs/README.md`. Having this documented within `docs/General` seems to defeat the purpose of the instructions.

Reply:
Eh, why? The purpose of the instructions is to tell anyone interested in adding to the documentation how to do it. Why should this be the very first thing people see when visiting? I'd hope that most visitors is visiting to read documentation, not just looking for how to add to it.